db/db_local/running_info.go

```diff
@@ -27,6 +28,17 @@ func (r *RunningDBInstanceInfo) Save() error {
 	return saveRunningInstanceInfo(r)
 }
 
+func (r *RunningDBInstanceInfo) String() string {
+	writeBuffer := bytes.NewBufferString("")
+	jsonEncoder := json.NewEncoder(writeBuffer)
+	p := r.Password
```

Comment:
add comment to explain what we are doing here

Reply:
added